### DISCUSSION TITLE
Work & Personal GitHub accounts, two licenses?

### DISCUSSION THREAD

**Discussion by skube** [community/community#19277] [Copilot Conversations] [answered]
Say I'm forced to use a work account (with a work email address) but I also have a personal account. 

Obviously, I'm only one person.

Must I pay double to use Copilot on both accounts?

**Comment by melvincwng** [comment]
@skube, to have access to GitHub Copilot on both accounts, you probably would have to pay twice.

**Comment by petersteele111** [comment]
I mean, you could just log into the one account that has co-pilot in your IDE and when pushing code to github do it a bit more manually and specify the login information and the repo you want it to go too. Here is a link that explains how to set the config for different accounts on a per repository basis. Would at least save you 10 dollars a month, though a bit more work and a bit more of a hassle. https://crunchify.com/how-to-set-github-user-name-and-user-email-per-repository-different-config-for-different-repository/

**Comment by justanotherkevin** [comment]
I have two accounts, one personal and one from work. I was about to make the purchase but realized that I would not be able it use it on my work GitHub account, which really sucks. I hope there is a workaround to this, but I also see the abuse that could come from allowing more than 1 GitHub account attached to a subscription.

**Comment by davecheney** [comment, marked as answer]
@skube afaik signing into vscode, so you can authenticate with copilot, has no effect on your git settings. If your employer requires you to do otherwise, I apologise. 

I want to also remind you that the terms and conditions for using copilot is as an _individual_, if you wish to use copilot for or on behalf of your employer, you should discuss that with them first.
